index.html

```diff
@@ -36,5 +36,137 @@ <h3 class="intro-2">LET'S CONNECT</h3>
+            </ul>
+            <button class="project-button">See project</button>
+        </div>
+    </section>
+    <section class="about-me">
+        <div>
+            <h2 class="about-header">About
+                Myself</h2>
+                <p class="about-paragraph">Hello I’m a software developer! I can <br>help you build a product , feature or <br> website Look through some of my <br>work and experience! If you like what you see <br>and  have a project you need coded,<br> don’t hestiate to contact me.</p>
+                <h3 class="about-intro">LET'S CONNECT</h3>
+                    <ul class="nav-links2">
+                        <li>
+                            <a href="#"><img src="images/github.png" alt="github"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="images/Linkedin.png" alt="linkedin"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img class="thumbs-up" src="images/thumbs.png" alt="thumbs-up"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="images/bird.png" alt="twitter"></a>
+                        </li> <li>
+                            <a href="#"><img src="images/mvs.png" alt="Vector"></a>
+                        </li>
+                    </ul>
+                    <button type="submit" class="resume-button">Get my resume</button>
+        </div>
+            
+                <div>
+                    <ul class="language">
+                        <li>
+                            <p class="language-paragraph">Languages</p>
+                        </li>
+                        <li><img src="images/down.png" alt="scroll-down"></li>
+                    </ul>
+    
+                        <ul class="language-list">
+                            <li class="list-items"><img src="images/Ellipse 1.png" alt="javascript">
+                                <p>javascript</p>
+                            </li>
+                            <li class="list-items" ><img src="images/Ellipse 1 (1).png" alt="html">
+                            <p>HTML</p></li>
+                            <li class="list-items"><img src="images/Ellipse 1 (2).png" alt="css"> <p>CSS</p>
+                            </li>
+                        </ul>
+                    
+                    <ul class="frameworks">
+                        <li>
+                            <p>Frameworks</p> </li>
+                        <li><img src="images/Disabled.png" alt="down"></li>
+                    </ul>
+                    <div class="line"></div>
+                    <ul class="gained-skill">
+                        <li>
+                            <p>Skills</p> </li>
+                        <li><img src="images/Disabled.png" alt="down"></li>
+                    </ul>
+                    <div class="line"></div>
+                </div>
```

Comment:
I like how you put the different languages into a list ie: ```JavaScript```, ```HTML``` and ```CSS``` 👍. How about you also consider putting ```Languages```, ```Frameworks``` and ```Skills``` in a list as well 😉. It will give your project a semantic touch. I'm available on slack mostly so do feel free to message me if this is not clear.